freshmaker/image_verifier.py

```diff
@@ -22,120 +22,119 @@
 # Written by Jan Kaluza <jkaluza@redhat.com>
 
 from freshmaker import conf
-from freshmaker.lightblue import LightBlue
+from freshmaker.pyxis_gql import PyxisGQL
 
 
 class ImageVerifier(object):
-
-    def __init__(self, lb=None):
+    def __init__(self, pgql=None):
         """
         Creates new ImageVerifier.
-
-        :param LightBlue lb: Lightblue instance to use to verify images.
-            When None, new default Lightblue class is created.
+        :param PyxisGQL pgql: PyxisGQL instance to use to verify images.
+            When None, new default PyxisGQL class is created.
         """
-        self.lb = lb if lb else LightBlue(
-            server_url=conf.lightblue_server_url,
-            cert=conf.lightblue_certificate,
-            private_key=conf.lightblue_private_key)
+        self.pgql = (
```

Comment:
could you rename the var to `self.pyxis` to keep it consistent with code in other modules?